### PR TITLE
use release-0.2

### PR DESCRIPTION
**Pull request by maleck13**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ plugins:
             - /README.md
             - /doc/*
         - name: multicluster-gateway-controller
-          import_url: 'https://github.com/kuadrant/multicluster-gateway-controller?edit_uri=/blob/main/&branch=main'
+          import_url: 'https://github.com/kuadrant/multicluster-gateway-controller?edit_uri=/blob/main/&branch=relase-0.2'
```
Comment (pmccarthy) [inline mkdocs.yml:64]: Typo in "release-0.2"

**Comment by maleck13** [comment]
@pmccarthy ok to re-review and we can get this merged

**Comment by pmccarthy** [comment]
Validated by running the outlined steps in the Getting Started guide:
```
export MGC_BRANCH=release-0.2
curl "https://raw.githubusercontent.com/kuadrant/multicluster-gateway-controller/${MGC_BRANCH}/hack/quickstart-setup.sh" | bash
```

**Pull request merged**